### PR TITLE
Convert to new upload flow

### LINKED ISSUE #5: Update with new upload endpoint

### PR DESCRIPTION
**Pull request by jasonaowen**
Rewrite the upload step of the functional test to use the new upload flow: get presigned URL, upload to S3, and then call register record.

Resolves #5 Update with new upload endpoint

### CODE REVIEW

File: run_test.py [hunk 1]
```diff
@@ -48,25 +47,96 @@ def make_request(self):
...
+    }]
+    request = PermanentRequest('/api/record/getPresignedUrl', data=body)
+    return request.response['SimpleVO']['value']
+
+
+def upload_to_s3(s3_url, file_path, presigned_post):
+    fields = presigned_post['fields']
+    with open(file_path, 'rb') as f:
+        fields['Content-Type'] = 'application/octet-stream'
+        logging.debug('Data to be form encoded for S3:')
```
Comment (mithuna) [inline run_test.py:66]: Do we need these logging statements?

File: run_test.py [hunk 2]
```diff
@@ -48,25 +47,96 @@ def make_request(self):
...
+        "parentFolder_linkId": parent_folder_link_id,
+        "displayName": filename,
+        "uploadFileName": filename,
+        "size": os.path.getsize(file_path),
+        "derivedCreatedDT": int(time.time()),
+    }
+
+    s3_info = get_presigned_url(record_vo)
+
+    logging.debug('S3 info:')
```
Comment (mithuna) [inline run_test.py:105]: Do we need these logging statements also?

**Review by mithuna — APPROVED** [review]
Test passed in local with exception of  couple of manual_review and converting. Attached report [https://docs.google.com/document/d/1Y3kLwVCSdQnyw5hTYo8-p5ICHHVzJzKJFcXHcDIurow/edit](url)

File: run_test.py (hunk 1 above)
Comment (jasonaowen) [reply run_test.py:66]: I think you're right that we could improve the logging statements!

I think there are some useful, debug-level logging statements that we should include, as these did help us during development, and I'll think a little more about what those should be.

Comment (jasonaowen) [reply run_test.py]: Added a new commit that cleans up the logging a bit. Thanks for the flag, @mithuna!

**Pull request merged**